src/app/archive/archive_lib/processor.ml

```diff
@@ -331,6 +331,28 @@ module User_command = struct
            |sql})
         id
 
+    type public_keys = {fee_payer_id: int; source_id: int; receiver_id: int}
```

Comment:
If I didn't add this type here it thought `{fee_payer_id; source_id; receiver_id}` below was referring to some other type and was missing required fields; is there a different workaround?